Makefile

```diff
@@ -8,9 +8,15 @@ all: main find-static
 main: main.cpp
 	$(CXX) -glldb $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions main.cpp -o ./main
 
+tool: tool.cpp
```

Comment:
Is it just me or is there no `tool.cpp` in this repository?

Reply:
Oh I forgot to remove that from when I was looking into libtooling.

Reply:
Removed it.